invokeai/app/util/get_timestamp.py

```diff
@@ -0,0 +1,5 @@
+import datetime
```

Comment:
This doesn't seem large enough for its own module. And if so, maybe the module should be `time.py` or something?

Reply:
done